pkg/standalone/uninstall.go

```diff
@@ -63,6 +63,20 @@ func removeDockerContainer(containerErrs []error, containerName, network, runtim
 	return containerErrs
 }
 
+func removeSchedulerVolume(containerErrs []error, runtimeCmd string) []error {
+	print.InfoStatusEvent(os.Stdout, "Removing volume if it exists: dapr_scheduler")
+	_, err := utils.RunCmdAndWait(
+		runtimeCmd, "volume", "rm",
+		"--force",
+		"dapr_scheduler")
+	if err != nil {
+		containerErrs = append(
```

Comment:
If the uninstall is for a version that does not have the volume at all, then it should just warn, just like in line 132.

Reply:
I suggest it to always warn.

Reply:
I made the change.

Reply:
Fair. I will undo it.

Reply:
I think we can revert to error then but only attempt to remove the volume if the scheduler is present? We should have the code to check for that. Otherwise the next best thing would be to only attempt removal if the version being uninstalled supports scheduler